projects/golang/go/1.18/patches/0002-syscall-expose-IfInfomsg.X__ifi_pad-on-s390x.patch

```diff
@@ -0,0 +1,41 @@
+From 67a4711d09c6595c17f32470c15bf471c287777d Mon Sep 17 00:00:00 2001
```

Comment:
102 and 101 are the same functionality -- we should re-use whichever patch came bundled with the Go 1.18 spec